### PR TITLE
EJ2-67758 : Instantiate PDF Viewer component dynamically and load PDF documents

### PR DESCRIPTION
**Pull request by SaravanaPriya31**
… documents

### CODE REVIEW

**Comment by SaravanaPriya31** [comment]
rebuild

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/Instantiate-pdfviewer-dynamically.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: Instantiate the PDF Viewer component dynamically and load PDF documents in ##Platform_Name## Pdfviewer Component
+description: Learn here all about Instantiate the PDF Viewer component dynamically and load PDF documents in Syncfusion ##Platform_Name## Pdfviewer component of Syncfusion Essential JS 2 and more.
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/Instantiate-pdfviewer-dynamically.md:4]: Description: Learn how to dynamically instantiate the PDF Viewer component and load PDF documents in the Syncfusion ##Platform_Name## Pdfviewer component of Syncfusion Essential JS 2 and more.

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/Instantiate-pdfviewer-dynamically.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+platform: ej2-asp-core-mvc
+control: Instantiate the PDF Viewer component dynamically and load PDF documents
+publishingplatform: ##Platform_Name##
+documentation: ug
+---
+
+# Instantiate the PDF Viewer component dynamically and load PDF documents
+
+To instantiate a PDF Viewer component dynamically and load PDF documents in Syncfusion PDF Viewer, you can use the following steps:
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/Instantiate-pdfviewer-dynamically.md:13]: To instantiate a PDF Viewer component dynamically and load PDF documents in Syncfusion PDF Viewer, use the following steps:

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/Instantiate-pdfviewer-dynamically.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+In your HTML file, create a div element that will act as a container for the PDF Viewer  component, and use the jQuery $.ajax method to make an AJAX request to your server and retrieve the PDF document. Once the document is retrieved, use the PDF Viewer API to load the document into the PDF Viewer component.
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/Instantiate-pdfviewer-dynamically.md:15]: In your HTML file, create a div element that will act as a container for the PDF Viewer component, and use the jQuery $.ajax method to make an AJAX request to your server and retrieve a PDF document. Once the document is retrieved, use the PDF Viewer API to load the document into the PDF Viewer component.

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/Instantiate-pdfviewer-dynamically.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+Use the serviceUrl property of the PDF Viewer instance to set the URL of the PDF document that you want to load.
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/Instantiate-pdfviewer-dynamically.md:17]: Use the serviceUrl property of the PDF Viewer instance to set the URL of a PDF document you want to load.

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/Instantiate-pdfviewer-dynamically.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+                });
+                viewer.appendTo("#pdfViewer");
+            }
+        });
+    });
+</script>
+
+```
+
+In your controller, create an action method that returns the PDF document as a FileResult.
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/Instantiate-pdfviewer-dynamically.md:44]: In your controller, create an action method that returns a PDF document as a FileResult.

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/Instantiate-pdfviewer-dynamically.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+Sample [https://www.syncfusion.com/downloads/support/directtrac/general/ze/InitiatePDFInButtonClick678733841.zip](https://www.syncfusion.com/downloads/support/directtrac/general/ze/InitiatePDFInButtonClick678733841.zip)
+
+Also, you can refer to the official documentation of syncfusion pdf viewer for more information [refer to this link](https://ej2.syncfusion.com/aspnetcore/documentation/pdfviewer/getting-started)
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/Instantiate-pdfviewer-dynamically.md:48]: Also, refer to the official documentation of syncfusion pdf viewer for more information [refer to this link](https://ej2.syncfusion.com/aspnetcore/documentation/pdfviewer/getting-started).

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/Instantiate-pdfviewer-dynamically.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+
+N>Ensure the provided document path and output text saved locations in your application level.
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/Instantiate-pdfviewer-dynamically.md:50]: N>Ensure the provided document path and output text are saved at your application level.

**Review by KameshRajendran — APPROVED** [review]

**Pull request merged**